Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/model/GameFile.java

```diff
@@ -72,9 +72,4 @@ public String getCoverPath(Context context)
   {
     return context.getExternalCacheDir().getPath() + "/GameCovers/" + getGameTdbId() + ".png";
   }
-
-  public String getCustomCoverPath()
-  {
-    return getPath().substring(0, getPath().lastIndexOf(".")) + ".cover.png";
-  }
```

Comment:
Like I mentioned before, please preserve the ability to set custom cover art for games.

Reply:
But I mentioned earlier that this does not work.

Reply:
I was under the impression it did work in some circumstances. Let me try to debug this in master.

Reply:
Fixed in PR #11017.

Even in master, it would work as long as you didn't use SAF. Setting up the game list without using SAF is a very hidden feature, though.